src/deployment/azuredeploy.json

```diff
@@ -17,6 +17,15 @@
         "signedExpiry": {
             "type": "string"
         },
+        "app_func_issuer": {
```

Comment:
These new arguments to the arm template need to be provided by `deploy.py`.  

This is where we currently set the arguments:
https://github.com/microsoft/onefuzz/blob/3096f99e863e7c1f660f7358b180ff56b1ba3df7/src/deployment/deploy.py#L400-L407

Reply:
Fixed.